modelbaker/utils/globals.py

```diff
@@ -34,3 +34,26 @@ class OptimizeStrategy(Enum):
     NONE = 0
     GROUP = 1
     HIDE = 2
+
+
+class LogLevel(IntEnum):
+    INFO = 0
+    WARNING = 1
+    FAIL = 2
+    SUCCESS = 3
+    TOPPING = 4
+
+
+def default_log_function(text, level=LogLevel.INFO, silent=False):
+    if silent:
+        return
+    if level == LogLevel.INFO:
+        print(f"INFO    : {text}")
```

Comment:
Couldn't we log instead to `QgsApplication.messageLog()` or you prefer the prints?

Reply:
It's because it might work without QGS